vpr/src/route/route_timing.cpp

```diff
@@ -1346,12 +1438,6 @@ static t_rt_node* setup_routing_resources(int itry,
             //we need to make sure the traceback is synchronized to the route tree
             traceback_from_route_tree(net_id, rt_root, reached_rt_sinks.size());
 
-            //Sanity check the traceback for self-consistency
-            VTR_ASSERT_DEBUG(validate_traceback(route_ctx.trace[net_id].head));
```

Comment:
Why did these get deleted?

Reply:
I don't remember deleting these. This may be the result of a wonky git merge.